src/connectors/scene/OtpSettingsSceneConnector.js

```diff
@@ -8,11 +8,16 @@ import type { Dispatch, State } from '../../modules/ReduxTypes'
 import OtpSettingsSceneComponent from '../../modules/UI/scenes/Otp/OtpSettingsSceneComponent.js'
 import * as SETTINGS_SELECTORS from '../../modules/UI/Settings/selectors.js'
 
-export const mapStateToProps = (state: State) => ({
-  isOtpEnabled: SETTINGS_SELECTORS.getIsOtpEnabled(state),
-  otpKey: SETTINGS_SELECTORS.getOtpKey(state),
-  otpResetDate: SETTINGS_SELECTORS.getOtpResetDate(state)
-})
+export const mapStateToProps = (state: State) => {
```

Comment:
I am fine with this, however it really doesn't change anything functionally

Reply:
Yeah it just helps for debugging